submissions/zhenyakornilov/js-dom/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/269a1f5e0e.js" crossorigin="anonymous"></script>
+  <link rel="icon" type="image/x-icon" href="images/museum-icon.svg">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Top Museums of Kyiv</title>
+</head>
+<body class="body">
+  <div class="wrapper">
+    <header class="header">
+      <span class="dropdown-wrap">
+        <button class="dropdown" id="dropdown">
+          <i class="dropdown-icon fa-solid fa-bars fa-lg"></i>
+        </button>
+      </span>
+      <h1 class="title">Top 5 Museums of Kyiv</h1>
+      <div>
+        <input class="check-switch" type="checkbox" id="switcher">
+        <label class="theme-switcher" for="switcher">
+          <i class="fa-solid fa-sun sun-icon"></i>
+          <i class="fa-solid fa-moon moon-icon"></i>
+          <span class="circle"></span>
+        </label>
+      </div>
+    </header>
+    <nav class="sidebar">
+      <ul class="menu-items" id="menu"></ul>
+    </nav>
+    <main class="content">
+      <article class="content-body" id="content"></article>
```

Comment:
I have misunderstanding here. I have **h2** tag added with JS inside **article** tag, isn't it can be counted as header? Or shall I set h2 tag inside **index.html**? May I just  change **article** tag to div then?

Reply:
you are right, I missed `h2`. All is fine here